provisioning/miniprov/hare_mp/main.py

```diff
@@ -223,6 +223,36 @@ def init(args):
         raise RuntimeError(f'Error while initializing cluster :key={error}')
 
 
+def start(args):
+    try:
+        path = os.getenv('PATH')
+        if path:
+            path += os.pathsep + '/opt/seagate/cortx/hare/bin'
+            path += os.pathsep + '/opt/seagate/cortx/hare/libexec'
+        python_path = os.pathsep.join(sys.path)
+        hare_log_file = '/var/log/seagate/hare/cortx-hare.log'
+        var_hax_path = '/var/motr/hax'
+        if not os.path.exists(var_hax_path):
+            os.makedirs(var_hax_path)
+        logger = logging.getLogger('hax-log')
+        fhandler = logging.handlers.RotatingFileHandler(hare_log_file,
+                                                        maxBytes=LOG_FILE_SIZE,
+                                                        mode='a',
+                                                        backupCount=5,
+                                                        encoding=None,
+                                                        delay=False)
+        logger.addHandler(fhandler)
+        cmd = ['hax']
```

Comment:
I think it is safer to execute hax script by the absolute path. In that case `PATH` env variable is not actually needed.

Reply:
Fixed the path related issue.